lib/gdtp_wrapper_impl.h

```diff
@@ -40,7 +40,7 @@ class gdtp_wrapper_impl : public gdtp_wrapper
     bool debug_;
     Addr src_address_;
     Addr dest_address_;
-    bool reliable_;
+    const std::vector<int> &reliable_;
```

Comment:
Yes, you are right, reference makes absolutely no sense. Maybe a copy and paste issue. :disappointed: